cmd/soroban-cli/src/commands/contract/alias/list.rs

```diff
@@ -0,0 +1,88 @@
+use std::fmt::Debug;
+use std::process;
+
+use clap::{command, Parser};
+
+use crate::commands::{config::network, global};
+use crate::config::locator;
+use crate::print::Print;
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub config_locator: locator::Args,
+
+    #[command(flatten)]
+    network: network::Args,
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Locator(#[from] locator::Error),
+
+    #[error(transparent)]
+    Network(#[from] network::Error),
+
+    #[error("no contract found with alias '{alias}' for network '{network_passphrase}'")]
+    NoContract {
+        alias: String,
+        network_passphrase: String,
+    },
+
+    #[error(transparent)]
+    PatternError(#[from] glob::PatternError),
+
+    #[error(transparent)]
+    GlobError(#[from] glob::GlobError),
+}
+
+impl Cmd {
+    pub fn run(&self, global_args: &global::Args) -> Result<(), Error> {
+        #[allow(unused_variables)]
+        let print = Print::new(global_args.quiet);
+        let network = self.network.get(&self.config_locator)?;
+        #[allow(unused_variables)]
+        let network_passphrase = &network.network_passphrase;
+        let config_dir = self.config_locator.config_dir()?;
+        let pattern = config_dir
+            .join("contract-ids")
+            .join("*.json")
+            .to_string_lossy()
+            .into_owned();
+
+        let paths = glob::glob(&pattern)?;
```

Comment:
If refactoring slightly we don't need to add a new dependency :)
```suggestion
        let paths = fs::read_dir(config_dir.join("contract-ids"))?
            .map(|res| res.map(|e| e.path()))
            .collect::<Result<Vec<_>, io::Error>>()?
            .into_iter()
            .filter(|e| e.extension() == Some("json".as_ref()));
```

Reply:
The dependency was already there (by another crate). I just made it explicit. :)